website/content/en/docs/migration/version-upgrade-guide.md

```diff
@@ -679,6 +681,109 @@ replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503
 - Run the command `operator-sdk generate k8s` to ensure that your resources will be updated
 - Run the command `operator-sdk generate crds` to regenerate CRDs
 
+**Bug Fixes and Improvements for Metrics**
```

Comment:
It is just an improvement. See: 

``` Improve skip metrics logs when running the operator locally in order to make clear the information. ([#2190](https://github.com/operator-framework/operator-sdk/pull/2190))```

Also, it is **OPTIONAL**. Users do not need to do this step to migrate their projects. 
We need to make clear it.

Reply:
Yes, but in 0.16.x we are referring to the function `addMetrics()` which is added here in 0.14.x, should the step from 0.16.x also be marked as optional?

https://sdk.operatorframework.io/docs/migration/version-upgrade-guide/#v016x

Reply:
I think it was added after with some patch release because it was done/released in 0.16. 
So, IMO here could have a note for the 0.16 instead of that. WDYT?

Reply:
Aha! now I got it. This commit became part of 0.14 https://github.com/operator-framework/operator-sdk/commit/81c6c8ae7e891971fcb234ee0ccd676e45f1d9d7, this means if anyone uses Operator SDK 0.14.x to create a project they will have `addMetrics()` in their project, right? (please correct me if I'm missing something here).

Now there are two possibilities, 

1. Someone is upgrading from a version prior to 0.14.x (like I was doing). They won't have addMetrics() when they are upgrading to 0.16.x, I had to go through the source code i.e. [the scaffold file](https://github.com/operator-framework/operator-sdk/blob/337264e75179df87e6d82659e957228c9e4606fe/internal/scaffold/cmd.go) to figure out how to replace whatever I had with those two functions i.e. the changes which I'm proposing as part of this PR for 0.14.x section.

2. Someone is upgrading from a version greater than 0.14.x, they will have the addMetrics() function and can upgrade easily by just following replace instructions from 0.16.x entry.

>So, IMO here could have a note for the 0.16 instead of that. WDYT?

Do you mean adding note in 0.16.x about addMetrics() and move the changes which I'm proposing for 0.14.x to 0.16.x? Something like "If you don't have addMetrics() then do this, if you have it then do that"? I think doing that will be better as the change was intended for 0.16.x.

Reply:
I catcher what do you mean. Marking it as solved it shows fine.